Build Fails on Windows 10 - Invalid RegEx in webpack.config

**Describe the bug**
Squoosh fails to build on Windows 10 because of an Invalid RegExp in webpack.config.js, issue could perhaps be related to #282 
**To Reproduce**
Steps to reproduce the behavior:
1. clone repo from master
2. run `npm run build` after `npm install`

generated error is as follows:
```
SyntaxError: Invalid regular expression: /\codecs\/: \ at end of pattern
    at new RegExp (<anonymous>)
    at module.exports (D:\Projects\squoosh\webpack.config.js:175:9)
    at process._tickCallback (internal/process/next_tick.js:68:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! squoosh@1.3.3 build: `webpack -p`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the squoosh@1.3.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Version:**
 - Windows 10
 - Node version: 10.13.10
 - npm version: 6.4.1

'npm run build' still fails with the error below (albeit different from the invalid regex error). These errors seem very similar to the problems originally reported in #282. I think that these lines (152 and 156) also need to make use of this path separator:
```javascript
test: /\/codecs\/.*\.js$/,
```
```javascript
test: /\/codecs\/.*\.wasm$/,
```
However, I know this isn't the full issue, because when I hard-code the windows file separator, the build still fails.

Current Error when attempting `npm run build`:
```
ERROR in ./src/codecs/processor-worker/index.ts (./node_modules/worker-plugin/dist/loader.js?{"name":"processor-worker"}!./src/codecs/processor-worker/index.ts)
Module build failed (from ./node_modules/worker-plugin/dist/loader.js):
ModuleParseError: Module parse failed: Unexpected character ' ' (1:0)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
    at handleParseError (D:\Projects\squoosh-google-original\squoosh\node_modules\webpack\lib\NormalModule.js:447:19)
    at doBuild.err (D:\Projects\squoosh-google-original\squoosh\node_modules\webpack\lib\NormalModule.js:481:5)
    at runLoaders (D:\Projects\squoosh-google-original\squoosh\node_modules\webpack\lib\NormalModule.js:342:12)
    at D:\Projects\squoosh-google-original\squoosh\node_modules\loader-runner\lib\LoaderRunner.js:370:3
    at iterateNormalLoaders (D:\Projects\squoosh-google-original\squoosh\node_modules\loader-runner\lib\LoaderRunner.js:211:10)
    at Array.<anonymous> (D:\Projects\squoosh-google-original\squoosh\node_modules\loader-runner\lib\LoaderRunner.js:202:4)
    at Storage.finished (D:\Projects\squoosh-google-original\squoosh\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:43:16)
    at provider (D:\Projects\squoosh-google-original\squoosh\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:79:9)
    at D:\Projects\squoosh-google-original\squoosh\node_modules\graceful-fs\graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)

ERROR in   Error: Child compilation failed:
  Module build failed (from ./node_modules/prerender-loader/dist/prerender-loader.js):
  Error: Child compilation failed:

  - prerender-loader.js:23
    [squoosh]/[prerender-loader]/dist/prerender-loader.js:23:31

  - Compiler.js:552 hooks.afterCompile.callAsync.err
    [squoosh]/[webpack]/lib/Compiler.js:552:14


  - after-compile.js:13
    [squoosh]/[ts-loader]/dist/after-compile.js:13:13


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compiler.js:549 compilation.seal.err
    [squoosh]/[webpack]/lib/Compiler.js:549:30


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1323 hooks.optimizeAssets.callAsync.err
    [squoosh]/[webpack]/lib/Compilation.js:1323:35


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1314 hooks.optimizeChunkAssets.callAsync.err
    [squoosh]/[webpack]/lib/Compilation.js:1314:32


  - index.js:318 taskRunner.run
    [squoosh]/[terser-webpack-plugin]/dist/index.js:318:9

  - TaskRunner.js:83 step
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:83:9

  - TaskRunner.js:94 done
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:94:30

  - TaskRunner.js:99 boundWorkers
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:99:13

  - TaskRunner.js:68 TaskRunner.boundWorkers
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:68:11

  - TaskRunner.js:89 enqueue
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:89:14

  - TaskRunner.js:109 tasks.forEach
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:109:9

  - Array.forEach

  - TaskRunner.js:87 TaskRunner.run
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:87:11

  - index.js:224 TerserPlugin.optimizeFn
    [squoosh]/[terser-webpack-plugin]/dist/index.js:224:18


  - Error: Child compilation failed:

  - compiler.js:79 childCompiler.runAsChild
    [squoosh]/[html-webpack-plugin]/lib/compiler.js:79:16

  - Compiler.js:296 compile
    [squoosh]/[webpack]/lib/Compiler.js:296:11

  - Compiler.js:552 hooks.afterCompile.callAsync.err
    [squoosh]/[webpack]/lib/Compiler.js:552:14


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compiler.js:549 compilation.seal.err
    [squoosh]/[webpack]/lib/Compiler.js:549:30


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1323 hooks.optimizeAssets.callAsync.err
    [squoosh]/[webpack]/lib/Compilation.js:1323:35


  - Hook.js:154 AsyncSeriesHook.lazyCompileHook
    [squoosh]/[tapable]/lib/Hook.js:154:20

  - Compilation.js:1314 hooks.optimizeChunkAssets.callAsync.err
    [squoosh]/[webpack]/lib/Compilation.js:1314:32


  - index.js:318 taskRunner.run
    [squoosh]/[terser-webpack-plugin]/dist/index.js:318:9

  - TaskRunner.js:44 TaskRunner.run
    [squoosh]/[terser-webpack-plugin]/dist/TaskRunner.js:44:7

  - index.js:224 TerserPlugin.optimizeFn
    [squoosh]/[terser-webpack-plugin]/dist/index.js:224:18


  - next_tick.js:68 process._tickCallback
    internal/process/next_tick.js:68:7



ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\generic\quality-option.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\generic\quality-option.tsx(3,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\imagequant\options.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\imagequant\options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\mozjpeg\options.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\mozjpeg\options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\optipng\options.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\optipng\options.tsx(6,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\resize\options.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\resize\options.tsx(6,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\webp\options.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\codecs\webp\options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\App\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\App\index.tsx(4,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Options\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Options\index.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\index.tsx(5,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\index.tsx(16,29)
      TS2307: Cannot find module './custom-els/TwoUp/styles.css'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\custom-els\TwoUp\index.ts
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\Output\custom-els\TwoUp\index.ts(2,25)
      TS2307: Cannot find module './styles.css'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\checkbox\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\checkbox\index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\compress\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\compress\index.tsx(5,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\compress\custom-els\MultiPanel\index.ts
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\compress\custom-els\MultiPanel\index.ts(1,24)
      TS2307: Cannot find module './styles.css'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\custom-els\LoadingSpinner\index.ts
./src/components/custom-els/LoadingSpinner/index.ts
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\custom-els\LoadingSpinner\index.ts(1,25)
      TS2307: Cannot find module './styles.css'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\expander\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\expander\index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\intro\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\intro\index.tsx(14,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\range\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\range\index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\results\FileSize.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\results\FileSize.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\results\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\results\index.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\select\index.tsx
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\components\select\index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\custom-els\RangeInput\index.ts
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\custom-els\RangeInput\index.ts(3,24)
      TS2307: Cannot find module './styles.css'.

ERROR in D:\Projects\squoosh-google-original\squoosh\src\lib\SnackBar\index.ts
[tsl] ERROR in D:\Projects\squoosh-google-original\squoosh\src\lib\SnackBar\index.ts(1,24)
      TS2307: Cannot find module './styles.css'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! squoosh@1.3.3 build: `webpack -p`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the squoosh@1.3.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Still fails, I've included the result of running npm install and npm build, in case there's something else that's the matter here:

```
                                           npm install
npm WARN @fimbul/ymir@0.15.0 requires a peer of tsutils@>2.29.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 44383 packages in 30.626s
found 65 vulnerabilities (64 low, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details

D:\Projects\squoosh-google-original\squoosh>npm run build

> squoosh@1.3.3 build D:\Projects\squoosh-google-original\squoosh
> webpack -p


No bundles were parsed. Analyzer will show only original module sizes from stats file.

Webpack Bundle Analyzer saved report to D:\Projects\squoosh-google-original\squoosh\build\report.html

Insufficient number of arguments or no entry found.
Alternatively, run 'webpack(-cli) --help' for usage info.


ERROR in Entry module not found: Error: Can't resolve 'src\index' in 'D:\Projects\squoosh-google-original\squoosh'
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! squoosh@1.3.3 build: `webpack -p`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the squoosh@1.3.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -142,11 +142,13 @@ module.exports = async function (_, env) {
         },
         {
           // All the codec files define a global with the same name as their file name. `exports-loader` attaches those to `module.exports`.
-          test: /\/codecs\/.*\.js$/,
+          test: /\.js$/,
+          include: path.join(__dirname, 'src/codecs'),
           loader: 'exports-loader'
         },
         {
-          test: /\/codecs\/.*\.wasm$/,
+          // For some reason using `include` here breaks the build.
+          test: /[/\\]codecs[/\\].*\.wasm$/,
           // This is needed to make webpack NOT process wasm files.
           // See https://github.com/webpack/webpack/issues/6725
           type: 'javascript/auto',
@@ -167,7 +169,7 @@ module.exports = async function (_, env) {
     plugins: [
       new webpack.IgnorePlugin(
         /(fs|crypto|path)/,
-        new RegExp(`${path.sep}codecs${path.sep}`)
+        /[/\\]codecs[/\\]/
       ),
 
       // Pretty progressbar showing build progress:

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,4 +1,3 @@
-const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
 const CleanPlugin = require('clean-webpack-plugin');
@@ -17,11 +16,7 @@ const CrittersPlugin = require('critters-webpack-plugin');
 const AssetTemplatePlugin = require('./config/asset-template-plugin');
 const addCssTypes = require('./config/add-css-types');
 
-function readJson (filename) {
-  return JSON.parse(fs.readFileSync(filename));
-}
-
-const VERSION = readJson('./package.json').version;
+const VERSION = require('./package.json').version;
 
 module.exports = async function (_, env) {
   const isProd = env.mode === 'production';
@@ -239,7 +234,7 @@ module.exports = async function (_, env) {
           removeRedundantAttributes: true,
           removeComments: true
         },
-        manifest: readJson('./src/manifest.json'),
+        manifest: require('./src/manifest.json'),
         inject: 'body',
         compile: true
       }),

